src/test/java/ebl/config/TestUtil.java

```diff
@@ -0,0 +1,52 @@
+package ebl.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+
+import java.io.*;
+import java.net.URL;
+import java.util.Map;
+
+public class TestUtil {
+
+  private static InputStream openStream(String resource) throws IOException {
+    URL url = Thread.currentThread().getContextClassLoader().getResource(resource);
+    if (url == null) {
+      throw new IllegalStateException("Cannot find json file " + resource);
+    }
+    return url.openStream();
+  }
+
+  public static String loadFileAsString(String resource) {
+    return parseResourceWithStream(
+        resource,
+        inputStream -> {
+          Reader dataInputStream = new BufferedReader(new InputStreamReader(inputStream));
+          StringBuilder stringBuilder = new StringBuilder();
+          char[] buffer = new char[4096];
+          while (dataInputStream.read(buffer) >= 1) {
+            stringBuilder.append(buffer);
```

Comment:
I think we need to use the `.append(buffer, 0, n)` version - otherwise we will append "garbage" if the read is less than 4096 bytes (which is very likely at the end of the file).